docs/root/start/sandboxes/local_ratelimit.rst

```diff
@@ -0,0 +1,95 @@
+.. _install_sandboxes_ratelimit:
+
+Local Ratelimit
+===============
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+        Used to make ``HTTP`` requests.
+
+Rate limiting is used to control the rate of requests sent or received by a network interface controller, which is helpful to prevent DoS attacks and limit web scraping.
+
+Envoy supports both local (non-distributed) and global rate limiting, and two types for local rate limiting:
+
+- L4 connections via the :ref:`local rate limit filter <config_network_filters_local_rate_limit>`
+- HTTP requests via the :ref:`HTTP local rate limit filter <config_http_filters_local_rate_limit>`
+
+This sandbox provides an example of rate limiting of L4 connections.
+
+Step 1: Start all of our containers
+***********************************
+
+Change to the ``examples/ratelimit`` directory and bring up the docker composition.
```

Comment:
this needs to be adjusted to the new filepath

Reply:
Done.